xla/service/gpu/llvm_gpu_backend/gpu_backend_lib_test.cc

```diff
@@ -33,6 +38,44 @@ TEST(UtilsTest, TestGetSmName) {
   ASSERT_EQ(nvptx::GetSmName(cc_next), "sm_90");
 }
 
+struct VersionPair {
+  int ptx_version;
+  int cuda_version;
+};
+using PtxVersionFromCudaVersionTest = ::testing::TestWithParam<VersionPair>;
+
+TEST_P(PtxVersionFromCudaVersionTest, VerifyMapping) {
+  std::array<int64_t, 3> tool_version{GetParam().cuda_version / 10,
+                                      GetParam().cuda_version % 10, 0};
+  EXPECT_EQ(GetParam().ptx_version,
+            nvptx::PtxVersionFromCudaVersionForTest(tool_version));
+}
+
+INSTANTIATE_TEST_SUITE_P(VersionTest, PtxVersionFromCudaVersionTest,
+                         ::testing::ValuesIn<VersionPair>({
+                             // CUDA 11
+                             {70, 110},
+                             {71, 111},
+                             {72, 112},
+                             {73, 113},
+                             {74, 114},
+                             {75, 115},
+                             {76, 116},
+                             {77, 117},
+                             {78, 118},
+                             // CUDA 12
+                             {80, 120},
+                             {81, 121},
+                             {82, 122},
+                             {83, 123},
+                             {84, 124},
+                             {85, 125},
+                             {85, 126},
```

Comment:
Nit: If you were reversing the order (CUDA version first, PTX version second), then it would feel more naturally like a mapping.

Reply:
Done.